pkg/app/destroy.go

```diff
@@ -58,7 +58,7 @@ func destroy(clicontext *cli.Context) error {
 		return errors.New("Cannot specify --path and --name at the same time.")
 	}
 	if path == "" && name == "" {
-		path = "."
+		return errors.New("Must specify --path or --name. If you want to destroy the current environment, please run `envd destroy -p .`")
```

Comment:
This will be a breaking change.

Personally, I prefer to confirm with users (if tty) if neither of them is provided. At least, it's faster than typing the name or path or waiting for shell completion.

BTW, we should print a log if the specific env doesn't exist. But the exit code could still be 0.

Reply:
Thank you for the prompt review. Confirming with user beforing deleteing the environment in `cwd` sounds more convenient while prevents breaking change. I am not sure what's the most standard way to display confirmation to tty. Could you provide me with some pointers? Would [pkg/app/interactive.go](https://github.com/tensorchord/envd/blob/main/pkg/app/interactive.go) be a good reference?

Reply:
Yeah, you can use that one. Or read from `os.Stdin` directly. Both work for me.

Reply:
Thank you for your input! Reading from `os.Stdin` sounds more readable and easy to implement. I'd go with that :)